app/deepseg.cc

```diff
@@ -213,11 +215,11 @@ class CalcMask final {
 	long maskns;
 	long loopns;
 
-	CalcMask(const char *modelname,
+	CalcMask(const std::string& modelname,
 			 size_t threads,
 			 size_t width,
 			 size_t height) {
-		maskctx = bs_maskgen_new(modelname,threads,width,height,nullptr,onprep,oninfer,onmask,this);
+		maskctx = bs_maskgen_new(modelname.c_str(),threads,width,height,nullptr,onprep,oninfer,onmask,this);
```

Comment:
Might be worth carrying over that type change to use STL types here too.

Reply:
Yep, will do.